### PR TITLE
Timestampable

### PR DESCRIPTION
**Pull request by Slavka-JD**

### CODE REVIEW

File: src/AppBundle/Entity/Performance.php [hunk 1]
```diff
@@ -5,14 +5,22 @@
 use Doctrine\ORM\Mapping as ORM;
 use Gedmo\Mapping\Annotation as Gedmo;
 use Symfony\Component\Validator\Constraints as Assert;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
```
Comment (spolischook) [inline src/AppBundle/Entity/Performance.php]: so, where is it?
also it needed to be named as "TimestampableTrait"

File: src/AppBundle/DataFixtures/ORM/LoadData.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?php
+namespace AppBundle\DataFixtures\ORM;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Hautelook\AliceBundle\Alice\DataFixtureLoader;
+use Nelmio\Alice\Fixtures;
+
+class LoadData extends DataFixtureLoader
+{
+    public function load(ObjectManager $om)
```
Comment (spolischook) [inline src/AppBundle/DataFixtures/ORM/LoadData.php]: remove this method

File: src/AppBundle/Entity/Employee.php [hunk 3]
```diff
@@ -224,4 +225,27 @@ public function getRoles()
     {
         return $this->roles;
     }
+
+    /**
+     * Get deletedAt
+     *
+     * @return \DateTime
+     */
+    public function getDeletedAt()
```
Comment (spolischook) [inline src/AppBundle/Entity/Employee.php]: this methods must be in trait

Comment (Slavka-JD) [inline src/AppBundle/Entity/Employee.php]: https://github.com/geekhub-php/CheTheatre/commit/5de95ef4d10641f7b4597d03984d24e3b29d79e5
done in this commit. in the final version in branch they are in trait

File: src/AppBundle/DataFixtures/ORM/LoadData.php (hunk 2 above)
Comment (Slavka-JD) [inline src/AppBundle/DataFixtures/ORM/LoadData.php]: removed in fixtures

**Comment by spolischook** [comment]
merge develop and fix conflicts

**Pull request merged**